imod/tests/test_prepare/test_topsystem.py

```diff
@@ -37,15 +37,21 @@ def test_riv_allocation(
         option, active, top, bottom, stage, bottom_elevation
     )
 
-    actual_riv = take_first_cell_in_xy_plane(actual_riv_da)
+    actual_riv = take_nth_cell_in_xy_plane(actual_riv_da, 0)
+    empty_riv = take_nth_cell_in_xy_plane(actual_riv_da, 1)
 
     if actual_drn_da is None:
         actual_drn = actual_drn_da
```

Comment:
actual_drn = None
is more explicit

Reply:
According to PEP8, ``is None`` is the preferred way to do things, as None is a singleton. See https://jaredgrubb.blogspot.com/2009/04/python-is-none-vs-none.html

Reply:
Oh sorry I misread your comment, that indeed makes the code more readable. I'll change that.